regression/pages/studio/unit_page.py

```diff
@@ -35,3 +35,35 @@ def add_word_cloud_component(self):
         self.wait_for_element_visibility(
             '.xblock-header-word_cloud', 'Word Cloud Component did not display'
         )
+
+    def add_lti_component(self):
+        """
+        Clicks Advanced button in Add New Component then
+        Clicks LTI and verifies that it appears on Studio
+        """
+        self.q(
+            css='.add-xblock-component-button[data-type="advanced"]'
+        ).click()
+        self.q(css='.button-component[data-category="lti"]').click()
+        self.wait_for_ajax()
+        self.wait_for_element_visibility(
+            '.xblock-header-lti', 'Word Cloud Component did not display'
```

Comment:
nit: the pattern is to say what we're waiting for, not treat it as an error message. This is because the error will show something like "Broken Promise: Word Cloud component displayed."